src/pages/registration/contributors_tab/Contributors.tsx

```diff
@@ -119,6 +130,9 @@ export const Contributors = ({ contributorRoles, push, replace }: ContributorsPr
       id: selectedContributor.id,
       name: getFullCristinName(selectedContributor.names),
       orcId: getOrcidUri(selectedContributor.identifiers),
+      verificationStatus: selectedContributor.verified
+        ? getVerificationStatus(selectedContributor.verified)
+        : VerificationStatus.CannotBeEstablished,
```

Comment:
Usiker på nøyaktig hva vi ønsker her. Men tolker det som at vi ønsker å sette `Verified` om `true`, `NotVerified` om `false`, og `CannotBeEstablished` om den mangler fra Cristin. Slik det står nå vil `CannotBeEstablished` settes om den er `false`.

```suggestion
      verificationStatus: getVerificationStatus(selectedContributor.verified)
```

Og endre `getVerificationStatus()` til:
```
export const getVerificationStatus = (verifiedStatus?: boolean) => {
  if (verifiedStatus === true) {
    return  VerificationStatus.Verified;
  } else if (verifiedStatus === false) {
    return  VerificationStatus.NotVerified;
  }
  return VerificationStatus.CannotBeEstablished;
};
```

Reply:
Å ja, for jeg trodde at den vil settes til `VerificationStatus.CannotBeEstablished` om den er undefined, endre nå med en gang.